gamemodes/terrortown/gamemode/shared/lang/english.lua

```diff
@@ -1278,3 +1278,21 @@ L.door_auto_closes = "This door closes automatically"
 L.door_open_touch = "Walk into door to open."
 L.door_open_touch_and_use = "Walk into door or press [{usekey}] to open."
 L.hud_health = "Health"
+
+-- 2020-04-20
+L.item_speedrun = "Speedrun"
+L.item_speedrun_desc = [[Makes you 50% faster!]]
+L.item_no_explosion_damage = "No Explosion Damage"
+L.item_no_explosion_damage_desc = [[Makes you immune to explosion damage.]]
+L.item_no_fall_damage = "No Fall Damage"
+L.item_no_fall_damage_desc = [[Makes you immune to fall damage.]]
+L.item_no_fire_damage = "No Fire Damage"
+L.item_no_fire_damage_desc = [[Makes you immune to fire damage.]]
+L.item_no_hazard_damage = "No Hazard Damage"
+L.item_no_hazard_damage_desc = [[Makes you immune to hazard damage such as posion, radiation and acid.]]
+L.item_no_energy_damage = "No Energy Damage"
+L.item_no_energy_damage_desc = [[Makes you immune to energy damage such as lasers, plasma and lightning.]]
+L.item_no_prop_damage = "No Prop Damage"
+L.item_no_prop_damage_desc = [[Makes you immune to prop damage.]]
+L.item_no_drown_damage = "No Drowning Damage"
+L.item_no_drown_damage_desc = [[Makes you immune to drowning damage.]]
```

Comment:
drowning or drown damage?

Reply:
I think `drowing` sounds better

Reply:
It is a little bit inconsistent then since all variable names use ```drown_damage``` .

Reply:
hm, I could rename the internal stuff as well, yes